### DISCUSSION TITLE
v0.9.0 - Mini Player Accents & Fixes

### DISCUSSION THREAD

**Discussion by Venipa** [Venipa/ytmdesktop2#42] [Announcements]
- Miniplayer fixes & layout update
- Miniplayer - Accent feature
- Updated Electron to v20

![image](https://user-images.githubusercontent.com/17952364/199159847-453cc854-4199-46bf-83ff-6f5f82a94df2.png)

**Full Changelog**: https://github.com/Venipa/ytmdesktop2/compare/v0.8.7...v0.9.0

<hr /><em>This discussion was created from the release <a href='https://github.com/Venipa/ytmdesktop2/releases/tag/v0.9.0'>v0.9.0 - Mini Player Accents & Fixes</a>.</em>

**Comment by hbuddz** [comment]
OMGGGG THANK YOU RANDOM PERSON FOR MAKING THIS!!

**Comment by E6WeB** [comment]
How to activcate the mini player?

**Reply by Venipa** [reply]
On the release page there's a preview of the mini player, see the icon on the top left?
That icon should be on the top right of the main window

**Reply by E6WeB** [reply]
I just downloaded the latest version and I do not see a icon for mini player at all.

**Reply by Venipa** [reply]
![screenshot.jpg](https://github.com/Venipa/ytmdesktop2/assets/17952364/7f3f71db-8a4d-48b9-86c2-026bb0f736e7)

Also make sure you downloaded from this GitHub or youtube-music.app
